nextflow.config

```
@@ -6,6 +6,8 @@
 ----------------------------------------------------------------------------------------
 */
 
+nextflow.enable.moduleBinaries = true
```

Just to make it clear 😄 
```suggestion
// Needed to use the R script for msnbasexic
nextflow.enable.moduleBinaries = true
```